src/polymer-cli.ts

```diff
@@ -165,6 +165,13 @@ export class PolymerCli {
     let command = this.commands.get(commandName);
     logger.debug('Running command...');
 
+    // Do any pre-run command initialization here, useful for setting up
+    // your command and any other expensive loading you don't want to do
+    // until neccessary.
+    if (command.init) {
```

Comment:
In a future PR I'd like to get rid of these startup-time optimizations that we are building into & maintaining inside of each command, and instead make the CLI smarter about how/when it loads commands (should only do so when needed). @justinfagnani let me know if you disagree with that direction.